server/args.py

```diff
@@ -0,0 +1,27 @@
+# args.py
+import argparse
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--sqlite-file-path",
+        default="pastes.db",
+        help=" The path to pastes.db",
```

Comment:
how about "path to the sqlite file to store paste metadata. defaults to ./pastes.db"